### PR TITLE
(RE-7514) Enable Bootlog and other Configuration Fixes

### PR DESCRIPTION
**Pull request by jcoconnor**
Apply a number of configuration updates to the image including:

1. [RE-7514](https://tickets.puppetlabs.com/browse/RE-7514) - Enable Bootlog
2. [RE-7589](https://tickets.puppetlabs.com/browse/RE-7514) - Apply windows update
3. [RE-7517](https://tickets.puppetlabs.com/browse/RE-7517) - Disable Windows Update
4. [RE-7523](https://tickets.puppetlabs.com/browse/RE-7523) - Configure WinRM
5. [RE-7683](https://tickets.puppetlabs.com/browse/RE-7683) - Disable NetBios and other services

### CODE REVIEW

**Comment by jcoconnor** [comment]
@glennsarti @McdonaldSeanp This is ready for review now - a collection of small configuration changes.

File: templates/windows-2012r2/files/config-vmware-vsphere-cygwin.ps1 [hunk 1]
```diff
@@ -9,6 +9,10 @@ $ErrorActionPreference = "Stop"
 # Although this is no longer run under boxstarter, we are still able to use it's cmdlets.
 Write-Host "Other Stuff......."
 
+# Enable Bootlog
+Write-Host "Enable Bootlog"
+cmd /c "bcdedit /set {current} bootlog yes"
```
Comment (glennsarti) [inline templates/windows-2012r2/files/config-vmware-vsphere-cygwin.ps1]: Possible nit, do you need the `cmd /c` in the front?

Comment (jcoconnor) [inline templates/windows-2012r2/files/config-vmware-vsphere-cygwin.ps1]: I think so - it failed in the script if I didn't have it.

Comment (jcoconnor) [inline templates/windows-2012r2/files/config-vmware-vsphere-cygwin.ps1]: Output from powershell:
```
PS C:\Users\Administrator> bcdedit /set {current} bootlog yes
The set command specified is not valid.
Run "bcdedit /?" for command line assistance.
The parameter is incorrect.
PS C:\Users\Administrator> cmd /c "bcdedit /set {current} bootlog yes"
The operation completed successfully.
PS C:\Users\Administrator>
```

Comment (glennsarti) [inline templates/windows-2012r2/files/config-vmware-vsphere-cygwin.ps1]: Ahh ok....Please ignore

**Comment by jcoconnor** [comment]
Closing as #94 supersedes this.

**Pull request closed without merging**